Motor_Handler.cpp

```diff
@@ -84,11 +89,20 @@ int16_t makePositive(int16_t x) {
   }
 }
 
-void Motor_Handler::handleWarningMessage(Frame& message) {
-  uint16_t warning_string = (message.body[2] << 8) + message.body[1];
+void Motor_Handler::handleStateMessage(Frame& message){
+  uint32_t State_string = (message.body[2]<<8) + (message.body[3]<<16) + (message.body[4]<<24) + message.body[1];
```

Comment:
variables should start with a lowercase letter, also same thing about the spacing around the << operators.